crates/core/migrations/2022-04-27-183240_create_graph_program_columns/down.sql

```diff
@@ -0,0 +1,3 @@
+alter table graph_connections;
+drop column connected_at;
```

Comment:
```suggestion
alter table graph_connections
drop column connected_at
```
Can you verify that this down migration actually works?

Reply:
Good catch! Yeah I'm still pending to do some more local testing.